cmd/ro/roagent/server.go

```diff
@@ -0,0 +1,274 @@
+// Copyright 2012 The Go Authors. All rights reserved.
```

Comment:
Is it?

Reply:
I don't follow. Should I have changed the copyright information? The licenses match and since this file for the most part is a copy of `x/crypto/ssh/agent/server.go` where I replaced the existing PrivateKey interfaces with one that contains a RO-encrypted private key, I don't think it counts a new code.